### DISCUSSION TITLE
DocFX v3 - Multilanguage, status, etc..

### DISCUSSION THREAD

**Discussion by UO258198** [dotnet/docfx#8168] [General]
I have been reading about DocFX v3 and have a few questions.

- Is there going to be a stable version?
- In the current betas is there support for multilanguage (Spanish, French, etc..). If not, is it planned?
- Finally, is there documentation for DocFX v3?

Thanks in advance

**Comment by VaclavElias** [comment]
I have been playing with docfx v3 (beta) for the last couple of days... It seems enough stable for me personally but I am using just basic functionality. The documentation can be found in the v3 branch https://github.com/dotnet/docfx/tree/v3, just check any/every .md file including in this folder https://github.com/dotnet/docfx/tree/v3/docs.

**Reply by UO258198** [reply]
And what about multilanguage support?

**Reply by VaclavElias** [reply]
Not sure, I am new here as you are but maybe this is still related to v3?

https://github.com/dotnet/docfx/blob/v3/docs/designs/conceptual-localization.md
